webapp/channels/src/components/external_image/external_image.tsx

```diff
@@ -23,7 +23,7 @@ const ExternalImage = (props: Props) => {
     if (!shouldRenderImage) {
         src = '';
     }
-    return props.children(src);
+    return (<>{props.children(src)}</>);
 };
 
-export default ExternalImage;
+export default memo(ExternalImage);
```

Comment:
I added back memo. Is there a script to update snapshots or do i need to manually edit it?

![Screenshot 2023-10-18 at 02 23 03](https://github.com/mattermost/mattermost/assets/66423127/fa103e22-9df0-4365-a085-30fd3bdf48b1)

I checked the mattermost channel. The only things that came up was https://community.mattermost.com/core/pl/s6bx66zgjinhfdndnwrwa18w7r

Reply:
You can run `npm run test -- -u -- path/to/test.test.ts`

Reply:
ty ty